content/id/docs/tasks/run-application/force-delete-stateful-set-pod.md

```diff
@@ -0,0 +1,81 @@
+
+   * Objek Node telah dihapus (baik oleh kamu, atau oleh [Pengontrol Node](/docs/admin/node)).<br/>
+   * Kubelet pada Node yang tidak responsif akan menanggapi, lalu mematikan Pod dan menghapusnya dari apiserver.<br/>
+   * Penghapusan paksa Pod oleh pengguna.
+
+Praktik terbaik yang direkomendasikan adalah menggunakan pendekatan pertama atau kedua. Jika sebuah Node telah terkonfirmasi mati (contohnya terputus dari jaringan secara permanen, dimatikan, dll), lalu menghapus objek Node. Jika Node mengalami partisi jaringan, maka coba selesaikan masalah ini atau menunggu masalah itu terselesaikan. Saat partisi terselesaikan, kubelet akan menyelesaikan penghapusan Pod serta membebaskan namanya dari apiserver.
+
+Normalnya, sistem akan menyelesaikan penghapusan saat Pod tidak lagi berjalan pada Node, atau Node telah dihapus oleh administrator. Kamu dapat mengabaikan hal ini dengan menghapus paksa Pod.
+
+### Penghapusan Paksa
+
+Penghapusan paksa **tidak** menunggu konfirmasi dari kubelet bahwa Pod telah diterminasi. Terlepas dari apakah penghapusan paksa sukses mematikan sebuah Pod, namanya akan segera dibebaskan dari apiserver. Hal ini berakibat pengontrol StatefulSet akan membuat Pod pengganti dengan identitas yang sama; ini dapat menimbulkan duplikasi terhadap Pod apabila ternyata Pod tersebut masih berjalan, dan jika Pod tersebut masih dapat berkomunikasi dengan anggota Statefulset lainnya, hal ini berakibat terjadi pelanggaran semantik _at most one_ dari StatefulSet yang telah dijamin.
+
+Saat kamu menghapus paksa sebuah Pod StatefulSet, berarti kamu menjamin bahwa Pod tersebut tidak akan pernah lagi berkomunikasi dengan Pod lain pada StatefulSet dan namanya dapat dibebaskan secara aman untuk pembuatan penggantinya.
+
+Jika kamu ingin menghapus paksa Pod dengan menggunakan kubectl versi >= 1.5, lakukan perintah berikut:
+
+```shell
+kubectl delete pods <pod> --grace-period=0 --force
+```
+
+Jika kamu menggunakan kubectl <= 1.4, kamu harus menghilangkan pilihan `--force` dan gunakan:
+
+```shell
+kubectl delete pods <pod> --grace-period=0
+```
+
+Jika setelah perintah ini dijalankan dan pod tetap berada pada kondisi `Unknown`, gunakan perintah berikut untuk menghapus pod dari klaster:
+
+```shell
+kubectl patch pod <pod> -p '{"metadata":{"finalizers":null}}'
+```
+
+Selalu jalankan penghapusan paksa Pod StatefulSet dengan hati-hati dan penuh pemahaman terhadap risiko yang dapat timbul.
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+Pelajari lebih lanjut [debugging StatefulSet](/docs/tasks/debug-application-cluster/debug-stateful-set/).
```

Comment:
```suggestion
Pelajari lebih lanjut [_debugging_ StatefulSet](/id/docs/tasks/debug-application-cluster/debug-stateful-set/).
```

Reply:
There is no ID localization yet for task debugging.